### DISCUSSION TITLE
Meeting schedule with Daylight Saving Time in Northern hemisphere

### DISCUSSION THREAD

**Discussion by tarilabs** [dmn-tck/tck#429] [General]
As previously anticipated in the previous monthly meeting, I am proposing a new monthly schedule, following Daylight Saving Time being now effective in the northern hemisphere,

FROM 
```
every 2nd Friday of each month.
The default time is 10am PST, 13:00 EST(Toronto), 18:00 GMT(London), 19:00 CET(Milano), 05:00amAEDT(Melbourne)
```

TO
```
every second Thursday of each month.
The default time be 08:00am EDT(Toronto), 13:00 BST(London), 14:00 CEST (Milano), 22:00 AEDT (Melbourne)
```

unfortunately with DST it is more difficult to find a good schedule covering worldwide, but as always feel free to counter-propose alternative schedule.

As a reminder, attending the meeting is OPTIONAL and we are striving to be as much inclusive to time differences and transparent in the proceedings, by making most of the work on the Pull Request and Milestones pages here on github.

**Comment by tarilabs** [comment]
* with the exception of the next one for the current month of April to be held on April 15th, 2021.

**Comment by StrayAlien** [comment]
That timing is fine by me @tarilabs - thank you.  Sure beats 3am. :-)

**Comment by tarilabs** [comment]
For the occurence of the TCK meeting on April 15th, I revised the open PRs and I believe we can realistically converge on these: https://github.com/dmn-tck/tck/milestone/3

I would seize the chance to discuss a bit about:
- `names` (as in DRG name / FEEL name) 
- typeRef for "invocables"

as I believe some clarification would help for some of other open PRs

**Comment by tarilabs** [comment]
For the occurence of the TCK meeting on May 13th, we should try to converge regarding:

- https://github.com/dmn-tck/tck/issues/430#issuecomment-829937842

Accordingly, a number of PRs can reflect and be merged accordingly if we can agree on the common ground suggested above:

- https://github.com/dmn-tck/tck/pull/436
- https://github.com/StrayAlien/tck/pull/3
- https://github.com/StrayAlien/tck/pull/4

then accordingly we should be able to finalize on:

- https://github.com/dmn-tck/tck/milestone/4

We might have as an opportunity to quickly discuss about:

- `names` (as in DRG name / FEEL name)

as I believe some clarification would help for some of other open PRs

**Comment by tarilabs** [comment]
For the occurence of the TCK meeting on June 10th, we should try to converge regarding:

https://github.com/dmn-tck/tck/pull/438

I believe it has a fundamental basis to potentially affect other work, so is best we check we're all on the same page for this. I will amend _this_ message to reflect if any additional items can be brought in.

**Reply by tarilabs** [reply]
I'd also like to review status on https://github.com/dmn-tck/tck/pull/393 as I was about to act on what we agreed in the meeting, but then I found all tests commented out? 🤔

**Reply by tarilabs** [reply]
we can also discuss about https://github.com/dmn-tck/tck/pull/394#issuecomment-857482342 time permitting.

**Reply by StrayAlien** [reply]
Hi @tarilabs I've closed 438, I good with it.  Rather than lose the hour on that topic, if we can I'd like to give more airtime to some of the as-yet unresolved issues like: https://github.com/dmn-tck/tck/issues/386, https://github.com/dmn-tck/tck/issues/376, and anything people feel is incorrect with the other PRs.  If we may.

**Reply by tarilabs** [reply]
Hi, okay I hear you, but I still believe it would be beneficial to summarize our understanding on #438 even if now is closed, so to valorise the effort we have spent there.

**Comment by tarilabs** [comment]
For the occurence of the TCK meeting on July 15th, I am currently collecting topics.

I've detected a potential (execution semantic) regression in non backward compatible change (1.2 -> 1.3) which I would prefer to iterate first in the TCK before _anyway_ presenting it at the OMG RTF.

**Reply by tarilabs** [reply]
https://github.com/dmn-tck/tck/pull/456#issuecomment-876969738

**Reply by tarilabs** [reply]
https://github.com/dmn-tck/tck/pull/422#issuecomment-877004141

**Comment by tarilabs** [comment]
cc: @opatrascoiu @StrayAlien @falko @SimonRinguette @vpellegrino

I am afraid there are problems with the email system of Circleweaver hence why tagged you on this thread. I am not even sure if a related https://github.com/dmn-tck/tck/discussions/472#discussion-3556981 reached you in the previous month.

The next scheduled meeting is:

```
14-Oct-2021
Shifted by default 1 our "earlier" in Northen emisphere to accomodate Australia shifted to DST.
Thursday 07:00am EDT(Toronto), 12:00 BST(London), 13:00 CEST(Milano), 22:00 AEDT(Melbourne).
```

in the Circleweaver calendar.

Please let me know if the email from Circleweaver did reach you, or otherwise; I will take into account for email/communications considerations.

I propose the following agenda:

- updates from DecisionCamp etc
- consider for merge previously passing tests and validator extension: https://github.com/dmn-tck/tck/milestone/7
- initial considerations on https://github.com/dmn-tck/tck/issues/470#issue-988500135 or other open issues
- define scope for next milestone

don't worry if due to email non-notified you can't join the live meeting, as usual we will proceed only on items having explicit consensus.

**Comment by opatrascoiu** [comment]
Hi Mateo,
I can't find the meeting invite details? Could you send them over please?
Thanks     On Tuesday, 12 October 2021, 22:15:05 BST, Matteo Mortari ***@***.***> wrote:  
 
 


cc: @opatrascoiu @StrayAlien @falko @SimonRinguette @vpellegrino

I am afraid there are problems with the email system of Circleweaver hence why tagged you on this thread. I am not even sure if a related #472 (comment) reached you in the previous month.

The next scheduled meeting is:
14-Oct-2021
Shifted by default 1 our "earlier" in Northen emisphere to accomodate Australia shifted to DST.
Thursday 07:00am EDT(Toronto), 12:00 BST(London), 13:00 CEST(Milano), 22:00 AEDT(Melbourne).

in the Circleweaver calendar.

Please let me know if the email from Circleweaver did reach you, or otherwise; I will take into account for email/communications considerations.

I propose the following agenda:
   
   - updates from DecisionCamp etc
   - consider for merge previously passing tests and validator extension: https://github.com/dmn-tck/tck/milestone/7
   - initial considerations on #470 (comment) or other open issues
   - define scope for next milestone

don't worry if due to email non-notified you can't join the live meeting, as usual we will proceed only on items having explicit consensus.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by tarilabs** [comment]
cc: @opatrascoiu @StrayAlien @falko @SimonRinguette @vpellegrino

I am afraid there are problems with the email system of Circleweaver hence why tagged you on this thread, also reason for having created this mailing-list on google group: https://groups.google.com/g/dmn-community/about you should have received invite to join it.

I propose the next scheduled meeting in line with North hemisphere not observing DST:

```
Friday 12 November ~every 2nd Friday of each month.
15:00 EST(Montréal), 20:00 GMT(London), 21:00 CET(Milano), 07:00amAEDT(Melbourne)
```

This appears to be the only possible schedule which does not include red slots by [source](https://www.timeanddate.com/worldclock/meetingtime.html?day=12&month=11&year=2021&p1=157&p2=152&p3=165&iv=0). Feel free to counter-propose schedule timings.

I propose the following agenda:
- anything finalised in https://github.com/dmn-tck/tck/milestone/8
- confirmation for https://github.com/dmn-tck/tck/pull/366#issuecomment-962993075
- https://github.com/dmn-tck/tck/issues/400 and related https://github.com/dmn-tck/tck/pull/483

No worry if for any reasons you can't join the live meeting, as usual we will proceed only on items having explicit consensus.

**Comment by tarilabs** [comment]
I propose the next scheduled meeting in line with North hemisphere not observing DST and on Tuesdat following :

```
Tuesday 14 December (Wed 15 in Australia)
15:00 EST(Montréal), 20:00 GMT(London), 21:00 CET(Milano), 07:00amAEDT(Melbourne)
This appears to be the only possible schedule which does not include red slots by source. Feel free to counter-propose schedule timings.
```

I propose the following agenda:

anything finalised in https://github.com/dmn-tck/tck/milestone/9

stretch goal, if _all_ the comments are actioned by then:
- https://github.com/dmn-tck/tck/pull/412

discussion on:
- https://github.com/dmn-tck/tck/issues/386#issuecomment-984489042
- https://github.com/dmn-tck/tck/issues/487#issuecomment-986600226

**Reply by opatrascoiu** [reply]
@tarilabs Hi Could you send over the meeting details please? Thanks

**Comment by tarilabs** [comment]
Happy new 2022 everyone!

🗓️ New recurrent meeting instance
A google calendar/vCal will be sent as usual through https://groups.google.com/g/dmn-community/about containing also the usual Google Meet link.

I propose the next scheduled meeting in line with North hemisphere not observing DST and on Tuesday following :

> Tuesday 11 January (Wed 12 in Australia) 15:00 EST(Montréal), 20:00 GMT(London), 21:00 CET(Milano), 07:00amAEDT(Melbourne) This appears to be the only possible schedule which does not include red slots by source. Feel free to counter-propose schedule timings.

I propose the following agenda:

anything finalised in https://github.com/dmn-tck/tck/milestone/10

meaning:

- https://github.com/dmn-tck/tck/pull/412
- (infra) https://github.com/dmn-tck/tck/pull/493

Stretch goals:

We(colleagues and me) have some additional comments about https://github.com/dmn-tck/tck/pull/390#pullrequestreview-830358131 which I count to submit by then

No action needed, but worth noting about https://github.com/dmn-tck/tck/discussions/488

Finally, we could start an informal poll about timing for planning the TCK migrating to DMNv1.4

No worry if for any reasons you can't join the live meeting, as usual we will proceed only on items having explicit consensus.
